library/Rules/Uuid.php

```diff
@@ -34,14 +37,16 @@ final class Uuid extends Standard
 {
     public const TEMPLATE_VERSION = '__version__';
 
-    private const PATTERN_FORMAT = '/^[0-9a-f]{8}-[0-9a-f]{4}-%s[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i';
-
     public function __construct(
         private readonly ?int $version = null
     ) {
+        if (!class_exists(RamseyUuid::class)) {
```

Comment:
This is the biggest changes, and it was my miscommunication before. I want this rule to require `ramsey/uuid`, but not the library to do so.